hashing/Cargo.toml

```diff
@@ -28,3 +28,5 @@ criterion = "0.3.5"
 proptest = "1.0.0"
 proptest-attr-macro = "1.0.0"
 tempfile = "3.1.0"
+serde_json = "1.0"
+rand = "0.8.4"
```

Comment:
Does your editor strip newlines at the end of files @rafal-ch ?

Reply:
No, it doesn't.